.github/workflows/solidity.yml

```diff
@@ -54,6 +54,7 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
       - run: yarn install --frozen-lockfile
+      - run: yarn workspace @chainlink/contracts compile
```

Comment:
Can we add this to the `./tools/ci/solidity_coverage` script instead?

Reply:
:+1: